go/engine/prove.go

```diff
@@ -313,20 +320,49 @@ func (p *Prove) promptPostedLoop(m libkb.MetaContext) (err error) {
 	return err
 }
 
+// Poll forever until the proof succeeds.
```

Comment:
forever?

Reply:
The context could get canceled. An hour?